src/atoms/Position/Position.native.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components/native';
+import { makePxValue } from '../../_helpers/theme';
+
+const Position = styled(({ children, style, ...props }) => {
+  if (React.Children.toArray(children).length !== 1) {
+    throw new Error('Expected a single child for Position component');
+  }
+  return React.cloneElement(children, {
+    ...props,
+    style: [style, children.props.style].filter(Boolean),
+  });
+})`
+  ${(props) => (props.position ? `position: ${props.position};` : '')};
+  ${(props) => (props.top ? `top: ${makePxValue(props.top)};` : '')};
+  ${(props) => (props.right ? `right: ${makePxValue(props.right)};` : '')};
+  ${(props) => (props.bottom ? `bottom: ${makePxValue(props.bottom)};` : '')};
+  ${(props) => (props.left ? `left: ${makePxValue(props.left)};` : '')};
+  ${(props) => (props.zIndex ? `z-index: ${props.zIndex};` : '')};
+`;
+
+Position.propTypes = {
+  position: PropTypes.oneOf(['relative', 'absolute']),
```

Comment:
One of - static | relative | absolute | sticky | fixed

Reply:
For native it should be only `relative` and `absolute`